### PR TITLE
Nicer output in console, fix example 05 and add python conf

### PR DESCRIPTION
**Pull request by ftence**

### CODE REVIEW

File: 05.mc2 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+$iter
+<%
+  // Convert the Row instance to a LIST
+  SPARK->
+  // Explode the LIST
+  LIST-> [ 'gts' 'ts' 'value' NULL ] STORE
+
+  // Add value to the encoder
+  $ts TOLONG NaN NaN NaN $value TOLONG ADDVALUE
```
Comment (hbs) [inline 05.mc2:27]: ts and value are no longer STRINGs

File: 01.py [hunk 2]
```diff
@@ -56,4 +56,9 @@
 rdd = sc.newAPIHadoopRDD('io.warp10.hadoop.Warp10InputFormat', 'org.apache.hadoop.io.Text', 'org.apache.hadoop.io.BytesWritable', conf=conf)
 df = rdd.toDF()
 
-print df.take(10)
+schemaTreeString = df._jdf.schema().treeString()
```
Comment (hbs) [inline 01.py:59]: This breaks the conciseness of the previous script, can you make it more compact?

File: 05.py [hunk 3]
```diff
@@ -31,7 +31,7 @@
 df.createOrReplaceTempView('GROUPED')
 sqlContext.registerJavaFunction("ws", "io.warp10.spark.WarpScriptUDF2", "BINARY")
 
-df = sqlContext.sql("SELECT ws('@04.mc2', _2) AS wrapper FROM GROUPED")
+df = sqlContext.sql("SELECT ws('@05.mc2', _2) AS wrapper FROM GROUPED")
```
Comment (hbs) [inline 05.py:34]: The loading should have been adapted from 04.py so the types are identical and therefore no need for a duplicate 05.mc2

**Comment by ftence** [comment]
Issues raised by this PR directly fixed by @hbs, closing.

**Pull request closed without merging**